Makefile

```diff
@@ -1,17 +1,13 @@
-mkfile_dir := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
-OUTPUT := $(mkfile_dir)output/.output
-CLANG ?= clang-13
-LLVM_STRIP ?= llvm-strip-13
-LIBBPF_SRC := /root/bcc/src/cc/libbpf
-LIBBPF_OBJ := $(abspath $(OUTPUT)/libbpf.a)
-INCLUDES := -I$(OUTPUT) -I$(LIBBPF_SRC)/include/uapi
+CLANG ?= clang
+LLVM_STRIP ?= llvm-strip
 CFLAGS := -g -O2 -Wall
 ARCH := $(shell uname -m | sed 's/x86_64/x86/' | sed 's/aarch64/arm64/' | sed 's/ppc64le/powerpc/' | sed 's/mips.*/mips/')
-
-GOFLAGS := CC=$(CLANG)
-GOFLAGS += CGO_CFLAGS="-I$(abspath $(OUTPUT))"
-GOFLAGS += CGO_LDFLAGS="-lelf -lz $(LIBBPF_OBJ) -lm"
-
+LIBBPFGO_DIR := $(abspath third_party/libbpfgo)
+LIBBPF_DIR := $(LIBBPFGO_DIR)/libbpf
+LIBBPF_OUTPUT := $(LIBBPFGO_DIR)/output
+LIBBPF_LIB := $(LIBBPF_OUTPUT)/libbpf.a
+GOFLAGS += CGO_LDFLAGS="-lz $(LIBBPF_LIB) -lm"
```

Comment:
This line needs to be : GOFLAGS += CGO_CFLAGS="-I$(LIBBPF_OUTPUT)" CGO_LDFLAGS="$(LIBBPF_LIB)"  - the zlib will come from the mbedded #cgo LDFLAGS directive in libbgfo.go , the -lm was originally required as I got some compilation errors in yara native code and seems to work now. We also need to make LIBBPF_OUPUT as an include path so that bpf/bpf.h can be found by libbpfgo as well as pass the direct path to the static libbpf object. With those changes I'm able to build velociraptor

Reply:
ack